identifier/src/containers/Login/Login.jsx

```diff
@@ -83,6 +83,10 @@ function Login(props) {
       history.replace(`/chooseaccount${history.location.search}${history.location.hash}`);
       return;
     }
+
+    if(query && query.login_hint){
```

Comment:
Space is missing after "if". Please use the same coding style as existing code.

Reply:
Updated with spaces and made consistent with the rest of code.